client/question/components/SpeechBalloonDirectives.js

```diff
@@ -0,0 +1,113 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for displaying speech balloons.
+ */
+
+tie.directive('tieSpeechBalloonContainer', ['$timeout', function($timeout) {
+  return {
+    restrict: 'A',
```

Comment:
Could we do `restrict: 'E'` instead? Better for directives to represent DOM elements, otherwise you introduce the possibility of one DOM tag having multiple of these attrs and that's confusing.

Ditto below.

Reply:
OK, will do.

Reply:
Done.